raiden/blockchain_events_handler.py

```diff
@@ -55,13 +56,15 @@ def handle_tokennetwork_new(raiden, event: Event):
         payment_network_identifier=event.originating_contract,
         token_network=token_network_state,
         block_number=block_number,
+        block_hash=block_hash,
     )
     raiden.handle_state_change(new_token_network)
 
 
 def handle_channel_new(raiden, event: Event):
     data = event.event_data
     block_number = data['block_number']
+    block_hash = views.blockhash_from_blocknumber(raiden.chain, block_number)
```

Comment:
Please use the data available from the event itself instead of doing another RPC call:

https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getfilterchanges

~~Additionally, events are handled before the `Block` state change, which may mean the data is not available yet (assuming this is introducing a way to map block numbers to block hashes through the state)~~

Reply:
Ah thanks man. Will do. I missed that the blockhash is already there.